currency-rates/internal/services/mail.go

```diff
@@ -1,42 +1,87 @@
 package services
 
 import (
+	"crypto/tls"
 	"net/smtp"
 
 	"go.uber.org/zap"
 )
 
+const serverName = "smtp.gmail.com"
 type Mail struct {
 	from string
 	pass string
+	client   *smtp.Client
 	l *zap.SugaredLogger
 }
 
-func NewEmail(from string, pass string, l *zap.SugaredLogger) *Mail {
+func NewEmail(from string, pass string, l *zap.SugaredLogger) (*Mail, error) {
+	client, err := smtp.Dial("smtp.gmail.com:587")
```

Comment:
Firstly,  server name that is used here (`smtp.gmail.com`) is already specified [here](https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-RinaDish/blob/hw2-solid/currency-rates/internal/services/mail.go#L10). 

Also server name and server port should be in environment config.

Reply:
Done